### PR TITLE
Update compute_hemodynamic_indices.py

### PR DESCRIPTION
**Pull request by ehsannkh**
Adding ECAP computation

### CODE REVIEW

**Comment by ehsannkh** [comment]
@hkjeldsberg I've added ECAP computation. Can you also test it ?

File: automatedPostProcessing/compute_hemodynamic_indices.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 except NameError:
     pass
 
+from IPython import embed
```
Comment (hkjeldsberg) [inline automatedPostProcessing/compute_hemodynamic_indices.py:13]: This can be removed 😁

**Comment by hkjeldsberg** [comment]
@ehsannkh Tested on the artery model, worked fine! Nice work! 

We should validated this implementation more properly with an atrium model with "known" outcome.

**Comment by ehsannkh** [comment]
> @ehsannkh Tested on the artery model, worked fine! Nice work!
> 
> We should validated this implementation more properly with an atrium model with "known" outcome.

@hkjeldsberg Totally agree. I will try to find a test for validation.

File: automatedPostProcessing/compute_hemodynamic_indices.py (hunk 1 above)
Comment (ehsannkh) [reply automatedPostProcessing/compute_hemodynamic_indices.py:13]: @hkjeldsberg Nice observation :)

**Pull request merged**